content/sql/concepts/subqueries/subqueries.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'SQL Subqueries' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Learn about SQL subqueries and how they can be used in different scenarios.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'SQL'
+
```

Comment:
```suggestion
  - 'Computer Science'
  - 'Data Science'
```
We generally do not create subjects for items that already have a topic dedicated to them